app/controllers/kuroko2/executions_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
   end
 
   def destroy
-    if @execution.try!(:pid)
+    if @execution.terminatable?
```

Comment:
Checking terminatable(L10) and creating an execution record(L13) are not atomic operations. Is it acceptable?

Reply:
added a unique index to prevent duplication